src/2023/index.html

```diff
@@ -12,7 +12,14 @@
   <body>
     <div>
       <header class="constrain">
-        <nav class="menu">
+        <nav class="menu top-menu">
+          <ul class="wrap">
+            <li><a href="talks">Talks</a></li>
+            <li><a href="#location">Location</a></li>
```

Comment:
Should this be without the anchor?

Reply:
Once we have a dedicated page for the venue, yes. For now, it links to further down on the same page.